src/fullcalendar/eventClick.js

```diff
@@ -94,17 +94,26 @@ function handleEventClick(event, store, router, route, window) {
  * @param {Window} window The window object
  */
 function handleToDoClick(event, store, route, window) {
-	if (!store.state.settings.tasksEnabled) {
-		if (!isPublicOrEmbeddedRoute(route.name)) {
-			showInfo(t('calendar', 'Please ask your administrator to enable the Tasks App.'))
-		}
+
+	if (isPublicOrEmbeddedRoute(route.name)) {
 		return
 	}
 
 	const davUrlParts = event.extendedProps.davUrl.split('/')
 	const taskId = davUrlParts.pop()
 	const calendarId = davUrlParts.pop()
-	const url = `apps/tasks/#/calendars/${calendarId}/tasks/${taskId}`
 
+	const deckAppPrefix = 'app-generated--deck--board-'
+	let url
+	if (calendarId.startsWith(deckAppPrefix)) {
+		const board = calendarId.substr(deckAppPrefix.length)
+		const card = taskId.substr('card-'.length).replace('.ics', '')
+		url = `apps/deck/#/board/${board}/card/${card}`
```

Comment:
@georgehrke Would you be fine with such an app related approach or should we just have a route at the deck app that takes the dav calendarId and taskId so that no parsing is done in calendar?

Reply:
Should we just use nc/event-bus here? Then any app can register custom handlers and calendar doesn't need to know anything about how their routers work.

Reply:
Sounds good to me.